schemas/mw-page-delete.yaml

```diff
@@ -0,0 +1,53 @@
+title: MediaWiki Page Delete
+description: Represents a MW Page Delete event
+type: object
+properties:
+  # global event fields
+  meta:
+    type: object
+    properties:
+      uri:
+        type: string
+        format: uri
+        description: the unique URI identifying the event
+      request_id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique UUID v1 ID of the event derived from the X-Request-Id header
+      id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique ID of this event; should match the dt field
+      dt:
+        type: string
+        format: date-time
+        description: the time stamp of the event, in ISO8601 format
+      domain:
+        type: string
+        description: the domain the event pertains to
+    required:
+      - uri
+      - request_id
```

Comment:
I'm wondering if request_id shouldn't be required in meta data.  It seems very possible that events will be created from sources that don't originate from an HTTP request.  E.g. what if the change propagation jobs need to insert a different type of event to trigger a different type of change?